activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -318,6 +318,19 @@ def rename_column(table_name, column_name, new_column_name) # :nodoc:
         rename_column_indexes(table_name, column.name, new_column_name)
       end
 
+      def add_timestamps(table_name, **options)
+        options[:null] = false if options[:null].nil?
+
+        if !options.key?(:precision) && supports_datetime_with_precision?
```

Comment:
`supports_datetime_with_precision` of SQLite3Adapter always returns true so we do not need ` && supports_datetime_with_precision?`
https://github.com/rails/rails/blob/aa00ef35eb0314c2b68103a7dfaf2d75a33cbc0b/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb#L154-L156

Reply:
Ah right, thanks! I've removed it.